src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/DoNotUseInsecureDtdProcessing.cs

```diff
@@ -186,6 +199,11 @@ public void AnalyzeOperationBlock(OperationBlockAnalysisContext context)
                 foreach (KeyValuePair<ISymbol, XmlDocumentEnvironment> p in _xmlDocumentEnvironments)
                 {
                     XmlDocumentEnvironment env = p.Value;
+
+                    // secure 4.5.2 version not being checked for secure
+                    // if Version > 4.5.2 => XmlDocument = secure!!
+                    // Before 4.5.2, insecure if XmlResolver set as null for XmlUrlResolver
+                    // After 4.5.2, insecure only if XmlResolver set as XmlUrlResolver
                     if (!(env.IsXmlResolverSet | env.IsSecureResolver))
```

Comment:
I'd remove the comments here, since I don't think they're matching the code here.

Reply:
Removed